datanode/bootstrap/bootstrap_server_test.go

```diff
@@ -58,6 +59,11 @@ var _ = ginkgo.Describe("bootstrap server", func() {
 	}
 
 	ginkgo.BeforeEach(func() {
+		testCreateTime := int64(1564646400)
```

Comment:
you can use  SetCurrentTime instead:
```
utils.SetCurrentTime(time.Unix(1564646400, 0))
```

Reply:
that is not expected,  we still need the time difference between calls

Reply:
I'm not following.. 
utils.Now() will always return time as represented by timestamp `1564646400` after your change.
which is the same as `utils.SetCurrentTime`'s behavior.

>> here are other calls to use the time functions
that's ok since we `utils.ResetClockImplementation()` after each test below.

Reply:
i mean we dont want to always return the fixed timestamp, we want to return the timestamp, plus the time elapsed between calls